### DISCUSSION TITLE
Welcome to vkr-mirea-project-management Discussions!

### DISCUSSION THREAD

**Discussion by kr4chinin** [kr4chinin/vkr-mirea-project-management#10] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kr4chinin** [comment]
Test discussions!

**Comment by kr4chinin** [comment]
Test discussions 2: Electric boogaloo!

**Comment by k0ndrateff** [comment, marked as answer]
Summer is coming!
